### PR TITLE
Post review changes

### PR DESCRIPTION
**Pull request by dgem**
Based on feedback from @galiri

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,80 +1,61 @@
...
+__An API for reading/writing brands into Neo4j. Expects the brands json supplied to be in the format that comes out of the brands extractor.__
 
-## Installation
-
-For the first time:
-
-`go get github.com/Financial-Times/brands-rw-neo4j`
-
-or update:
+[Runbook for service](https://sites.google.com/a/ft.com/ft-technology-service-transition/home/run-book-library/draftbrandsreadwritemicroservicedanmurphyftcom20160215)
```
Comment (sjwells) [inline README.md]: You can (and should) amend the url for the run book. I think it's in Manage Site for that page on the google site.

File: README.md [hunk 2]
```diff
@@ -1,80 +1,61 @@
...
-Ping: [http://localhost:8080/ping](http://localhost:8080/ping) or [http://localhost:8080/__ping](http://localhost:8080/__ping)
+* Healthchecks: [http://localhost:8080/__health](http://localhost:8080/__health)
+* Ping: [http://localhost:8080/ping](http://localhost:8080/ping) or [http://localhost:8080/__ping](http://localhost:8080/__ping)
 
 ### Loading Data
-_*Disclamimer:* this is still a work in progress_
-* You will need to install Ruby (2.0+) and the [Nokogiri](http://www.nokogiri.org/) and JSON libraries
-  `gem install nokogiri json`
-  Be aware that nokogiri uses the libxml2 library and so does some complex compilation, which I've not tried on Windows
+_*Disclamimer:* this is still a work in progress (the version of WPAT is not in prod)_
```
Comment (sjwells) [inline README.md]: Disclaimer (typo)

File: brands/brands_service_test.go [hunk 3]
```diff
@@ -11,49 +11,71 @@ import (
...
+}
+
+func TestCreateNotAllValuesPresent(t *testing.T) {
+	err := getCypherDriver(t).Write(validSkeletonBrand)
+	assert.NoError(t, err)
+	readBrandAndCompare(validSkeletonBrand, t)
+	cleanUp(validSkeletonBrand.UUID, t)
 }
 
 func TestDeleteExitingBrand(t *testing.T) {
```
Comment (sjwells) [inline brands/brands_service_test.go]: Existing?

File: main.go [hunk 4]
```diff
@@ -62,7 +62,7 @@ func makeCheck(service baseftrwapp.Service, cr neoutils.CypherRunner) v1a.Check
 	return v1a.Check{
 		BusinessImpact:   "Cannot read/write brands via this writer",
 		Name:             "Check connectivity to Neo4j - neoUrl is a parameter in hieradata for this service",
-		PanicGuide:       "TODO - write panic guide",
+		PanicGuide:       "https://sites.google.com/a/ft.com/ft-technology-service-transition/home/run-book-library/draftbrandsreadwritemicroservicedanmurphyftcom20160215",
```
Comment (sjwells) [inline main.go]: NB: Change when you change the url

File: README.md (hunk 1 above)
Comment (dgem) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (dgem) [inline README.md]: done

File: main.go (hunk 4 above)
Comment (dgem) [inline main.go]: done

File: brands/brands_service_test.go (hunk 3 above)
Comment (dgem) [inline brands/brands_service_test.go]: Lol... done

**Pull request merged**